### ISSUE TITLE
gain plugin not working in kX 3553

### ISSUE THREAD

**Issue by AwkwardAardvark** [kxproject/kx-audio-driver#20]
When adding Basic -> gain plugin and opening its settings for the first time, the following message appears:
 
![shot-20180901t125025](https://user-images.githubusercontent.com/42889188/44944921-06a08080-ade8-11e8-8c36-de563353f0fd.png)

Then the gain value is fixed at -174.60:

![shot-20180901t125109](https://user-images.githubusercontent.com/42889188/44944926-1b7d1400-ade8-11e8-89fd-aaceeb00bd69.png)

Plugin source as a *.da file is attached in the zip below.
[gain.zip](https://github.com/kxproject/kx-audio-driver/files/2342273/gain.zip)

**Comment by AwkwardAardvark** [comment]
Seems I have posted this in the wrong place (should have been in binaries instead). Is there a way to move the ticket?

**Comment by kxproject** [comment]
I don't think it is possible to move the ticket. However, kx-audio-driver (this repository) looks like a good place for such bug reports, let it stay here. Thanks!
